mozilla_django_oidc/views.py

```diff
@@ -114,15 +114,15 @@ def redirect_url(self):
 
     def dispatch(self, request, *args, **kwargs):
         """Log out the user."""
-
-        if request.user.is_authenticated():
+        # Check if a method exists to build the URL to log out the user
+        # from the OP.
+        logout_from_op = import_from_settings('OIDC_OP_LOGOUT_URL_METHOD', '')
```

Comment:
Actually. Why not? Maybe. 
What's a good example of an implementation of `OIDC_OP_LOGOUT_URL_METHOD`? Does it assume that the user *is* authenticated. 

Another option altogether is to do something like this:

```python
    def dispatch(self, request, *args, **kwargs):
        """Log out the user."""
        if not request.user.is_authenticated():
            return HttpResponseBadRequest('already signed out')
        logout_from_op = import_from_settings('OIDC_OP_LOGOUT_URL_METHOD', '')
        if logout_from_op:
            logout_url = import_string(logout_from_op)()
        else:
            logout_url = self.redirect_url
        auth.logout(request)
        return HttpResponseRedirect(logout_url)           
```

What do you think about that solution?